spec/unit/facter/simplib__firewalls_spec.rb

```diff
@@ -0,0 +1,15 @@
+require 'spec_helper'
+
```

Comment:
What about iptables in the test?

Reply:
@lnemsick-simp It's just one from Linux, one from Solaris, and one from BSD.

The `pf` is specifically to make sure the command is translated properly.

Not too overly concerned in checking that hashes work if the data is correct.

Reply:
What is the purpose of the fact then?  Can't you just set the firewall type data?